mpi-proxy-split/Makefile

```diff
@@ -45,6 +45,11 @@ endif
 ifndef DMTCP_ROOT
   DMTCP_ROOT=${MANA_ROOT}/dmtcp
 endif
+
+ifeq ($(DESTDIR),)
```

Comment:
Let's not touch DESTDIR at all -- this is something that's supplied as part of the make command itself.